servers/src/main/webapp/META-INF/log4j.xml

```diff
@@ -18,11 +18,9 @@
 	</appender>
 
 	<logger category="org.springframework">
-		<level name="INFO" />
 	</logger>
 
 	<root>
-		<priority value="INFO" />
 		<appender-ref ref="CONSOLE" />
```

Comment:
are these supposed to be removed ?

Reply:
Yes, in order to not hard-code log levels.

Reply:
so once removed the logs would be controlled through standalone.xml ? 

Are there any modifications we need to prepare for when upgrading already existing sites such as production?

Reply:
No, it's transparent and control is moved to the .xml as it should be.

Reply:
Cool